src/Uno.UI/UI/Xaml/UIElement.skia.cs

```diff
@@ -46,11 +46,6 @@ public UIElement()
 			UpdateHitTest();
 		}
 
-		~UIElement()
```

Comment:
`UIElement.wasm.cs` has something similar. I think the new code will be applicable to Wasm and we can remove the UIElement finalizer on Wasm as well?

Reply:
Wasm needs to keep its finalizer for now, as the purpose is different (clearing native resources).